.circleci/config.yml

```diff
@@ -158,7 +158,12 @@ jobs:
           name: Install dependencies
           command: |
             apt-get install -y wget jq make gettext gzip
+            apt install -y python3.11-venv
+            VENV_DIR="$(pwd)/.venv"
+            python3 -m venv $VENV_DIR
+            source "${VENV_DIR}/bin/activate"
             pip3 install yq==2.11.0
```

Comment:
The installation of `yq` had failed because CircleCI or the image was no longer allowing system-level installs of python packages this way.